### DISCUSSION TITLE
Accessing icon from protected bin repo

### DISCUSSION THREAD

**Discussion by romandolinsky** [chocolatey/choco#3450] [Q&A]
### Checklist

- [X] I have verified this is the correct repository for opening this issue.
- [X] I have verified no other issues exist related to my request.

### Describe The Solution. Why is it needed?

Our companies move to clouds solutions and privates nuget repos protected by credentials.  Is against security and long time support strategy downloading data from 3rd party pages. All our icons are store in antifactory in bin repo. Is there way to downloading icons by using Apikeys already stored in choco  ?  

### Additional Context

Downloading icons using apikeys. 

### Related Issues

https://github.com/chocolatey/chocolatey-licensed-issues/issues/346

**Comment by pauby** [comment]
This isn't a licensed issue, and is a question. Moving to a discussion.

@romandolinsky if you are Licensed customer, can you please run `choco support` to find out your support options and reach out to them?
